src/Jenkins.ts

```diff
@@ -88,7 +88,12 @@ export class Jenkins {
       let result: JenkinsStatus;
       
       request
-        .get(url + '/api/json')
+        .get(url + '/api/json', {
+          'auth': {
```

Comment:
@umens Thanks for checking this out.

I didn't test against 1.647 secured; that's a good idea.  I will secure my 1.647 instance and test against it later this morning.  I'll report back once I've done so.